### DISCUSSION TITLE
Is the format of artifacts.json guaranteed stable for the current version of Goreleaser?

### DISCUSSION THREAD

**Discussion by ArnoSen** [goreleaser/goreleaser#3750] [Ideas, Issue Triage, and General discussions]
Hi,

I would like to use artifacts.json for a post release action. I would like to know if the format of this file is kept unchanged  for this version of Goreleaser. 

If yes, may I suggest to version the file so external tools consuming artifacts.json can exit nicely when it does not support particular versions of artifacts.json?

**Comment by caarlos0** [comment]
we don't break it on purpose, and don't change it often either (I don't think it ever changed, actually), but its not considered a guaranteed thing.

to add a version field to artifacts.json I think we would have to break it, as if I recall, the top level element is an array, and we'd need it to be an object I think
